cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/media/3Tdisk/ns3-TON/ns3/ns-3.14.1/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/media/3Tdisk/ns3-TON/ns3/ns-3.14.1/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 46506704: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-O3\', \'-g\', \'-Wall\', \'-Werror\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
